sbom-utility-scripts/scripts/index-image-sbom-script/index_image_sbom_script.py

```diff
@@ -107,11 +119,11 @@ def create_sbom(
     image_index_digest: str,
     inspect_input: dict[str, Any],
 ) -> dict:
-    if inspect_input["mediaType"] != "application/vnd.oci.image.index.v1+json":
+    if inspect_input["mediaType"] not in INDEX_IMAGE_MANIFEST_MEDIA_TYPES:
```

Comment:
Could you extract the mediaType changes into a separate commit? They seem unrelated to the SBOM name changes